app/src/main/java/com/lagradost/cloudstream3/utils/TextUtil.kt

```diff
@@ -1,17 +1,13 @@
-package com.lagradost.cloudstream3.ui.result
```

Comment:
Is there a reason behind this package move? It is a breaking change, so it needs to be a good reason.

Reply:
not encountered any issue, the file name should be textutil instead of uitext as image functions have been removed, location was also in wrong package